povcalnet.ado

```diff
@@ -258,8 +259,26 @@ qui {
 		}
 	}
 	
-	*---------- Poverty line
-	if ("`povline'" == "") local povline = 1.9
+	*---------- Poverty line/population share
+	if ("`popshare'" != ""){
+		if ("`povline'" != ""){
+			oi disp as err "povline and popshare cannot be used at the same time"
+			error
+		}
+		else{
+			loc povline = ""
+			loc pcall = "popshare"
+		}
+	}
+	else if ("`povline'" == "") {

```

Comment:
This condition is not mutually exclusive with the first condition and with the else statement.  I think the problem is that these lines, 
```stata
if ("`popshare'" != ""){
		if ("`povline'" != ""){
```
Should be just one line

Reply:
Agree, I'll adjust it